src/python/build_management/build_manager.py

```diff
@@ -1426,7 +1425,10 @@ def setup_custom_binary(target_weights=None):
 def setup_production_build(build_type):
   """Sets up build with a particular revision."""
   # Bail out if there are not stable and beta build urls.
-  if build_type == 'stable':
+  if build_type == 'extended_stable':
+    build_bucket_path = environment.get_value(
+        'EXTENDED_STABLE_BUILD_BUCKET_PATH')
```

Comment:
Can you `return None` here for now if this is None? Otherwise we'll log_error below all the time (resulting in email alerts).

Reply:
Done.